domains/org.eclipse.lyo.tools.domainmodels/oslcDomainSpecifications.xml

```diff
@@ -13,6 +13,9 @@
     <resourceProperties id="_pnXOMpUdEeq-KoPaR9_Cfg" name="description" description="Descriptive text about resource represented as rich text in XHTML content. SHOULD include only content that is valid and suitable inside an XHTML &lt;div> element." occurs="zeroOrOne" valueType="XMLLiteral">
       <propertyDefinition href="vocabulary.xml#//@vocabularies[label='Dublin%20Core']/@properties[name='description']"/>
     </resourceProperties>
+    <resourceProperties id="_fdNWAFv1EeyHYL9SbPncDg" name="description" description="Descriptive text about resource represented as rich text in XHTML content. SHOULD include only content that is valid and suitable inside an XHTML &lt;div> element." occurs="zeroOrOne" valueType="String">

```

Comment:
@jamsden haha here is where the weasel-wording bit us! Jena and Java do not care about our SHOULD (be valid XHTML) and throw exceptions quite happily if you feed them with bad XML.